packages/dashboard/src/hooks/useKeyPress.ts

```diff
@@ -56,9 +56,13 @@ export const useKeyPress = (key: string, options?: KeyPressOptions | KeyPressCal
   );
 
   useEffect(() => {
+    window.addEventListener('copy', onKeyPress as EventListener);
+    window.addEventListener('paste', onKeyPress as EventListener);
     window.addEventListener('keydown', onKeyPress);
```

Comment:
do we still need keyup and keydown ?

Reply:
yes, we need keydown and keyup listeners since isHotkey supports only **KeyboardEvent** but copy and paste are **ClipboardEvent**, due to this if we remove keydown and keyup then isHotkey returns false.